src/docs/PML/PMLUserGuide.adoc

```diff
@@ -4741,7 +4750,8 @@ NOTE: Please click https://peps.cnes.fr/rocket/#/register[here^] to create a new
 
 NOTE: Please click https://scihub.copernicus.eu/dhus/#/self-registration[here^] to create a new user account from Copernicus website.
 
-WARNING: Note que as variaveis `TIME_TO_RETRIEVE_IN_SECONDS`, `TIME_TO_CHECK_ONLINE_IN_SECONDS` and `WALLTIME` are valid only for the Copernicus platform.
+WARNING: Note que as variaveis `TIME_TO_RETRIEVE_IN_SECONDS`, `TIME_TO_CHECK_ONLINE_IN_SECONDS` and `WALLTIME` are valid only for the Copernicus platform. In addition, it is necessary to define a policy used by the ProActive Scheduler, in this case only two tasks must be executed simultaneously.
```

Comment:
valid only => only valid

Reply:
, in this case only two tasks must be executed simultaneously. => . In this case, only two tasks must be executed simultaneously.